seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/org/apache/seata/spring/boot/autoconfigure/loader/SeataPropertiesLoader.java

```diff
@@ -74,8 +77,23 @@ public void initialize(ConfigurableApplicationContext applicationContext) {
             environment.getPropertySources().addLast(new PropertiesPropertySource("seataOldConfig", properties));
         }
         // Load by priority
-        System.setProperty("sessionMode", StoreConfig.getSessionMode().getName());
-        System.setProperty("lockMode", StoreConfig.getLockMode().getName());
+        loadSessionAndLockModes();
     }
 
+    public void loadSessionAndLockModes() {
+        try {
+            Class<?> storeConfigClass = Class.forName("org.apache.seata.server.store.StoreConfig");
+            String sessionMode = invokeStaticMethod(storeConfigClass, "getSessionMode").orElse("defaultSessionMode");
+            String lockMode = invokeStaticMethod(storeConfigClass, "getLockMode").orElse("defaultLockMode");
```

Comment:
what is the `defaultSessionMode` and `defaultLockMode` ?

Reply:
I removed it